cdap-app-fabric/src/main/java/io/cdap/cdap/app/guice/AppFabricServiceRuntimeModule.java

```diff
@@ -267,6 +270,7 @@ public Module getDistributedModules() {
                            new ConfigStoreModule(),
                            new EntityVerifierModule(),
                            new ProvisionerModule(),
+                           new ProgramRunDispatcherModule(),
```

Comment:
now I'm confused, I thought you said the factory existed because we want to return an implementation based on the cconf. That seems to no longer be the case?